scanner/templates/node/Dockerfile

```
@@ -1,38 +1,53 @@
-FROM debian:bullseye as builder
+# syntax = docker/dockerfile:1
 
-ENV PATH=/usr/local/node/bin:$PATH
 ARG NODE_VERSION={{ .nodeVersion }}
+FROM node:${NODE_VERSION}-slim as base
```

I guess we can combine these?

```suggestion
FROM node:{{ .nodeVersion }}-slim as base
```

Still thinking we should make `.nodeVersion` a major version instead of a patch version if it doesn't come from `.nvmrc` (or any of the other mentioned files)

---

> [ARG](https://docs.docker.com/engine/reference/builder/#arg)s can be overridden at built time, and we've used them elsewhere to give a hint as to things that can be adjusted.

Didn't know this.
Let's keep all `ARG`s at the top of the file then 👍

---

Unfortunately, Docker is weird.  Anything mentioned by the first FROM statement needs to be above that statement.  Everything else needs to be either inside either the build stage where it is referenced, or inside a stage from which it is based.

---

Weird flex 😅